src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -58,7 +58,7 @@ public void InitializeAppInfo(string installedLocation)
 
             IStream stream;
             const uint noAttribute = 0x80;
-            const Stgm exclusiveRead = Stgm.Read | Stgm.DenyWrite;
+            const Stgm exclusiveRead = Stgm.Read | Stgm.DenyNone;
```

Comment:
We can reduce this to `const Stgm exclusiveRead = Stgm.Read` since `DenyNone`is default.

Reply:
Yes, you're right. Shall remove it because it is default.